test/e2e/placement_selecting_resources_test.go

```diff
@@ -387,7 +387,7 @@ var _ = Describe("validating CRP when cluster-scoped and namespace-scoped resour
 	It("should update the selected resources on member clusters", checkIfPlacedWorkResourcesOnAllMemberClusters)
 
 	It("should update CRP status as expected", func() {
-		crpStatusUpdatedActual := crpStatusUpdatedActual(workResourceIdentifiers(), allMemberClusterNames, nil)
```

Comment:
It could be either 1 or 2. It depends on the when our controller detects the changes.

Could you please update the test to update the resources one by one?

eg, update one resource -> validate -> update another resource -> validate